From a435f3c9318935ba29b04dabe7342cd6160711e0 Mon Sep 17 00:00:00 2001
From: El Mehdi YOUNES <elmehdi.younes@smile.fr>
Date: Thu, 15 May 2025 10:02:12 +0200
Subject: [PATCH] support out of tree build

prepare_builtins tool is built
but not placed in the path for later use in the build. This
fix allows the later build steps to use the binary in-place.

Upstream: Not submitted

Signed-off-by: El Mehdi YOUNES <elmehdi.younes@smile.fr>
---
 libclc/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d651a9bf506b..18bd7efe8217 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -180,7 +180,7 @@ set(LLVM_LINK_COMPONENTS
 )
 if( LIBCLC_STANDALONE_BUILD )
   add_llvm_executable( prepare_builtins utils/prepare-builtins.cpp )
-  set( prepare_builtins_exe prepare_builtins )
+  set( prepare_builtins_exe ./prepare_builtins )
   set( prepare_builtins_target prepare_builtins )
 else()
   add_llvm_utility( prepare_builtins utils/prepare-builtins.cpp )
-- 
2.43.0

